p2p/src/config.rs

```diff
@@ -101,6 +102,7 @@ pub struct Libp2pConfig {
     pub notify_handler_buffer_size: usize,
     pub connection_event_buffer_size: usize,
     pub dial_concurrency_factor: u8,
+    pub sql_db_path: PathBuf,
```

Comment:
See previous comment, I don't think we want this field here. Instead I would rather we just pass an instance of the blockstore down into the deps instead of passing a duplicate config path and creating a new connection.